dart_io_mini_samples/example/files_directories_and_symlinks/directories/creating_a_directory.dart

```diff
@@ -8,11 +8,8 @@
 
 import 'dart:io';
 
-void main() {
+main() async {
   // Creates dir/ and dir/subdir/.
-  new Directory('dir/subdir').create(recursive: true)
-    // The created directory is returned as a Future.
-    .then((Directory directory) {
-      print(directory.path);
-  });
+  var directory = await new Directory('dir/subdir');
```

Comment:
Shouldn't this be .create(recursive: true) so it'll create dir, as well as subdir?